app/features/environment_variables/environment_variables_controller.rb

```diff
@@ -42,21 +42,19 @@ class EnvironmentVariablesController < AuthenticatedControllerBase
           value: params[:value]
         )
 
-        success = Services.environment_variable_service.update_environment_variable!(
+        Services.environment_variable_service.update_environment_variable!(
           environment_variable: environment_variable
         )
-        logger.error("Something went wrong") unless success
```

Comment:
We log the error if this doesn't succeed [here](https://github.com/fastlane/ci/compare/master...AndrewMcBurney:remove_return_true_in_crud_actions?expand=1#diff-040760123e1c4871883a4e55898a005bR58).